library/core/src/ptr/mut_ptr.rs

```diff
@@ -2454,6 +2569,7 @@ mod verify {
     generate_mut_arithmetic_harness!((i8, i8), check_mut_sub_tuple_1, sub);
     generate_mut_arithmetic_harness!((f64, bool), check_mut_sub_tuple_2, sub);
     generate_mut_arithmetic_harness!((i32, f64, bool), check_mut_sub_tuple_3, sub);
+    generate_mut_arithmetic_harness!((i8, u16, i32, u64, isize), check_mut_sub_tuple_4, sub); 
```

Comment:
duplicated line

Reply:
Line removed in newest commit.

Reply:
Still adds whitespace at the end of the line.